src/Umbraco.AuthorizedServices/AuthorizedServicesComposer.cs

```diff
@@ -38,9 +42,26 @@ public void Compose(IUmbracoBuilder builder)
 
         builder.Services.AddUnique<ISecretEncryptor>(factory =>
             {
+                ILogger<AuthorizedServicesComposer> logger = factory.GetRequiredService<ILogger<AuthorizedServicesComposer>>();
                 var tokenEncryptionKey = configSection.GetValue<string>(nameof(AuthorizedServiceSettings.TokenEncryptionKey));
 
-                return new AesSecretEncryptor(tokenEncryptionKey ?? string.Empty);
+                if (string.IsNullOrWhiteSpace(tokenEncryptionKey))
+                {
+                    logger.LogWarning($"No encryption key was found in configuration at {ConfigurationRoot}:{nameof(AuthorizedServiceSettings.TokenEncryptionKey)}. Falling back to using the Umbraco:CMS:{nameof(GlobalSettings)}:{nameof(GlobalSettings.Id)} value.");
+
+                    IOptions<GlobalSettings> globalSettings = factory.GetRequiredService<IOptions<GlobalSettings>>();
+                    tokenEncryptionKey = globalSettings.Value.Id;
+
+                    return new NoopSecretEncryptor();
```

Comment:
This line looks like it will prevent `Umbraco:CMS:Global:Id` from being used for the encryption key as it uses the `NoopSecretEncryptor` right away.

Reply:
Thanks (head is already on the beach)... will fix up.

Reply:
Awesome. Didn't expect you to look at anything now, just discovered it when I had a look through the changes.